cypress/e2e/scenario-2.3-embedded-entity.cy.js

```diff
@@ -0,0 +1,249 @@
+
+    // expect row to be expanded
+    cy.get(".pf-c-table__expandable-row-content").should("to.be.visible");
+
+    // Expect to find status tab
+    cy.get(".pf-c-tabs__list li:first").should("have.class", "pf-m-current");
+
+    cy.get(".pf-c-description-list").contains("Diagnose").click();
+
+    // Diagonse sub-page should open and be empty
+    cy.get("h1").contains("Diagnose Service Instance").should("be.visible");
+    cy.get('[aria-label="Diagnostics-Empty"]').should("be.visible");
+    cy.visit("/console/");
+  });
+
+  it("2.3.3 - Show history view", () => {
+    cy.intercept(
+      "GET",
+      "lsm/v1/service_inventory/embedded-entity-service?include_deployment_progress=True&limit=20&&sort=created_at.desc"
+    ).as("GetServiceInventory");
+    cy.visit("/console/");
+    cy.get('[aria-label="Environment card"]').contains("lsm-frontend").click();
+    // Expect to find one badge on the embedded-service row.
+    cy.get("#embedded-entity-service")
+      .get('[aria-label="Number of instances by label"]')
+      .children()
+      .should("have.length", 1);
+    cy.get("#embedded-entity-service").contains("Show inventory").click();
+    cy.get("#expand-toggle0").click();
+
+    // expect row to be expanded
+    cy.get(".pf-c-table__expandable-row-content").should("to.be.visible");
+
+    // Expect to find status tab
+    cy.get(".pf-c-tabs__list li:first").should("have.class", "pf-m-current");
+    //await for instance state to change to up
+    checkServiceState();
+
+    cy.get(".pf-c-description-list").contains("History").click();
+
+    // History sub-page should open and be empty then go to Home page
+    cy.get("h1").contains("Service Instance History").should("be.visible");
+
+    // due to lack of Id in rows I had to assert that each toggle button is separeate history log
+    cy.get(".pf-c-table").find(".pf-c-table__toggle").should("have.length", 3);
+    cy.visit("/console/");
+  });
+
+  it("2.1.4 Delete previously created instance", () => {
+    cy.visit("/console/");
+
+    // Add interceptios for the delete and get call to be able to catch responses later on.
+    cy.intercept(
+      "DELETE",
+      "/lsm/v1/service_inventory/embedded-entity-service/**"
+    ).as("DeleteInstance");
+
+    cy.get('[aria-label="Environment card"]').contains("lsm-frontend").click();
+    cy.get("#embedded-entity-service").contains("Show inventory").click();
+
+    // wait to check if compile is done
+    waitForCompile();
+    cy.wait("@GetServiceInventory");
+
+    // expand first row
+    cy.get("#expand-toggle0").click();
+
+    // There might be still a compile running because of the edit action. So we do a check if one is pending.
+    waitForCompile();
+    cy.wait(2000);
+
+    // delete but cancel deletion in modal
+    cy.get(".pf-c-description-list").contains("Delete").click();
+    cy.get(".pf-c-modal-box__title-text").should("contain", "Delete instance");
+    cy.get(".pf-c-form__actions").contains("No").click();
+
```

Comment:
I had to add a cy.wait(2000) here in between because sometimes Cypress seems too fast and it will end up not finding the delete button because the modal is still pending; or at least that's what I understood. When the modal shows, the button will be disabled too, so when trying to click on it, it will not be able to because the state change is still occuring.

Reply:
I didn't came across this issue locally, I'll check on jenkins if this is happening there. Thanks for the tip

Reply:
on Jenkins also went through, so I don't know if I should add it

Reply:
It did break on the basic-service, but I don't know if the upgraded timeout resolved the flakeyness here or not